### PR TITLE
[16.0][MIG] mgmtsystem_nonconformity: Migration to 16.0

### PR DESCRIPTION
**Pull request by FernandoRomera**

### CODE REVIEW

File: mgmtsystem_nonconformity/views/mgmtsystem_action.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (max3903) [inline mgmtsystem_nonconformity/views/mgmtsystem_action.xml:1]:
```suggestion
```

There is no need to add those lines. Thanks!

Comment (FernandoRomera) [reply mgmtsystem_nonconformity/views/mgmtsystem_action.xml:1]: The documentation says: 
https://www.odoo.com/documentation/16.0/es/developer/reference/backend/data.html
for all XML files.

Comment (pedrobaeza) [reply mgmtsystem_nonconformity/views/mgmtsystem_action.xml:1]: On contrary, Maxime, they must be there. XML format specifies that. If they are not there, they are working by chance, and there's the possibility of a future version of `lxml` (or the low-level library used by Odoo in the future) doesn't parse it correctly. There can be also problems with certain characters if the encoding is not specified.

**Review by ivantodorovich — APPROVED** [review]

**Review by AntoniRomera — APPROVED** [review]

**Comment by FernandoRomera** [comment]
@max3903 
Can you merge this PR, please?

File: mgmtsystem_nonconformity/views/mgmtsystem_action.xml (hunk 1 above)
Comment (max3903) [reply mgmtsystem_nonconformity/views/mgmtsystem_action.xml:1]: Ok

**Pull request merged**